sql/catalyst/src/main/scala/org/apache/spark/sql/types/StructField.scala

```diff
@@ -118,6 +118,17 @@ case class StructField(
     }
   }
 
+  /**
+   * Return the existence default value of this StructField.
+   */
+  def getExistenceDefaultValue(): Option[String] = {
```

Comment:
Also should probably protect this via `private[sql]` because `StructField` is an API.